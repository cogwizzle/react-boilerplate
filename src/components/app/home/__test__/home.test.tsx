import * as React from 'react'
import Home from '../home'
import * as renderer from 'react-test-renderer'

it('Renders', () => {
  const component = renderer.create(
    <Home />
  )
  const results = component.toJSON()
  expect(results).toMatchSnapshot()
})

