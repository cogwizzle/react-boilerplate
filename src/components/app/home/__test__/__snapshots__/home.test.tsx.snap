// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div
  className="home"
>
  <h3>
    Welcome to your new React-Boilerplate project!
  </h3>
  <p>
    To learn more about the capabilities of React-Boilerpalte go to the documentation on github.
  </p>
</div>
`;
